mlserver/settings.py

```diff
@@ -87,6 +87,10 @@ class Config:
     model_repository_root: str = "."
     """Root of the model repository, where we will search for models."""
 
+    model_repository_implementation: Optional[PyObject]
+
+    model_repository_implementation_args: dict = {}
```

Comment:
As an extra note on the settings, would it be possible to add a comment below each of them? That will ensure it shows up on the generated docs.

Reply:
sure, I'll do it

Reply:
Done